gtsam/geometry/Rot2.cpp

```diff
@@ -129,6 +129,16 @@ Rot2 Rot2::relativeBearing(const Point2& d, OptionalJacobian<1, 2> H) {
   }
 }
 
+/* ************************************************************************* */
+Rot2 Rot2::ClosestTo(const Matrix2& M) {
```

Comment:
I applaud the existence of this method!
But not sure this is right. I think it should be done with SVD as in SO3, then c and s can just be read off as below.